app/controllers/state_file/questions/df_return_sample.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<Return xmlns="http://www.irs.gov/efile" xmlns:efile="http://www.irs.gov/efile" returnVersion="2023v5.0">
+<Return xmlns="http://www.irs.gov/efile" xmlns:efile="http://www.irs.gov/efile" returnVersion="2023v2.0">
```

Comment:
[dust] i don't think i remember or can find any place where we actually use `returnVersion` from the df fixtures. is there a reason to change this?